src/model/invite.rs

```diff
@@ -404,3 +422,33 @@ impl RichInvite {
         format!("https://discord.gg/{}", self.code)
     }
 }
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[cfg(feature = "model")]
+#[non_exhaustive]
+pub struct InviteStageInstance {
+    /// The members speaking in the Stage
+    members: Vec<PartialMember>,
+    /// The number of users in the Stage
+    participant_count: u64,
+    /// The number of users speaking in the Stage
+    speaker_count: u64,
+    /// The topic of the Stage instance (1-120 characters)
+    topic: String,
+}
+
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq, PartialOrd, Ord)]
+#[non_exhaustive]
+#[cfg(feature = "model")]
```

Comment:
The `model` feature only applies to methods on model structs. It shouldn't be present on structs nor enums.
```suggestion
```